src/c_commands.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include <unordered_map>
 
+#include <stdlib.h> // exit
```

Comment:
I prefer to use the C++ standard here - `cstdlib` instead of `stdlib.h`.  But see below.